core/templates/pages/about-page/about-page-root.component.spec.ts

```diff
@@ -49,17 +63,45 @@ describe('About Root Page', () => {
     fixture = TestBed.createComponent(AboutPageRootComponent);
     component = fixture.componentInstance;
     pageHeadService = TestBed.inject(PageHeadService);
+    translateService = TestBed.inject(TranslateService);
   });
 
   it('should successfully instantiate the component',
     () => {
       expect(component).toBeDefined();
     });
 
-  it('should initialize', () => {
-    spyOn(pageHeadService, 'updateTitleAndMetaTags');
+  it('should initialize and subscribe to onLangChange', () => {
+    spyOn(translateService.onLangChange, 'subscribe');
+    component.ngOnInit();
```

Comment:
```suggestion
    spyOn(translateService.onLangChange, 'subscribe');

    component.ngOnInit();
```

Reply:
Please add a line before the function is called. Please do this for all the tests.

Reply:
Hey, I've made the required changes but I'm having trouble pushing them. I recently moved my oppia directory to elsewhere and I think that's what's causing issues. I'm looking into it and will push the changes soon.

Reply:
Done, I'll keep this in mind!